release-please-config.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "include-component-in-tag": true,
+  "include-v-in-tag": true,
+  "tag-separator": "@",
+  "separate-pull-requests": true,
+  "extra-files": [
+    "README.md",
+    "mkdocs.yml"
+  ],
+  "packages": {
+    "backend/graph-proxy": {
+      "release-type": "rust"
+    },
```

Comment:
We should update the `appVersion` in the corresponding charts `Chart.yaml` when we release a new container, this can be [done with extra-files](https://github.com/googleapis/release-please/blob/main/docs/customizing.md#updating-arbitrary-yaml-files)

Reply:
done